web/src/engine/websites/LegacyScans.ts

```diff
@@ -19,19 +32,25 @@ export default class extends DecoratableMangaScraper {
     public override get Icon() {
         return icon;
     }
-}
 
-// Original Source
-/*
-class LegacyScans extends WordPressMangastream {
-
-    constructor() {
-        super();
-        super.id = 'legacyscans';
-        super.label = 'Legacy-Scans';
-        this.tags = ['webtoon', 'french'];
-        this.url = 'https://legacy-scans.com';
-        this.path = '/manga/list-mode/';
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        const mangaList = [];
+        let run = true;
+        let start = 1;
+        let end = start + this.mangasPerPages;
+        while (run) {
+            const mangas = await this.getMangasFromRange(provider, start, end);
+            mangas.length > 0 ? mangaList.push(...mangas) : run = false;
+            start = end + 1;
+            end = start + this.mangasPerPages;
+        }
+        return mangaList;
     }
-}
-*/
\ No newline at end of file
+    async getMangasFromRange(provider: MangaPlugin, start: number, end : number): Promise<Manga[]>{
```

Comment:
Is the `end` parameter necessary, since it can be inferred from `start` and `this.mangasPerPages`?